scripts/zones/Grauberg_[S]/Zone.lua

```diff
@@ -26,9 +27,9 @@ end
 
 function onZoneWeatherChange(weather)
     local npc = GetNPCByID(ID.npc.INDESCRIPT_MARKINGS)
-    if (weather == tpz.weather.WIND or weather == tpz.weather.GALES) then
+    if (weather == tpz.weather.WIND or weather == tpz.weather.GALES) and ENABLE_WOTG = 1 then
```

Comment:
all of these = conditional statements need to be =='s

     ``if (weather == tpz.weather.WIND or weather == tpz.weather.GALES) and ENABLE_WOTG == 1 then``